org.mozilla.Thunderbird.json

```diff
@@ -0,0 +1,92 @@
+{
+    "app-id": "org.mozilla.Thunderbird",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "thunderbird",
+    "rename-icon": "thunderbird",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* OpenGL access */
+        "--device=dri",
+        /* Needs to talk to the network: */
+        "--share=network",
+        /* Play sounds */
+        "--socket=pulseaudio",
+        /* Access to thunderbird configuration and cache files */
+        "--filesystem=~/.cache/thunderbird", "--filesystem=~/.thunderbird",
+        /* Access to home directory (for attachments) */
+        "--filesystem=home",
+        /* Needed for dconf to work */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "*.la", "*.a"
+    ],
+    "modules": [
+        {
+            "name": "autoconf-2.13",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnu.org/gnu/autoconf/autoconf-2.13.tar.gz",
+                    "sha256": "f0611136bee505811e9ca11ca7ac188ef5323a8e2ef19cffd3edb3cf08fd791e"
+                }
+            ],
+            "post-install": [
+	        "ln -s /app/bin/autoconf /app/bin/autoconf-2.13"
+            ]
+        },
+	{
+	    "name": "icu",
```

Comment:
icu also ships some files in /usr/bin that aren't needed for thunderbird to work. Any reason for using 58? 60 has been released some time ago.

Reply:
Mostly because This was someone elses project and I don't know much about how thunderbird is packaged, ignorance on my part...apologies.

Reply:
Going to test the build using 60, changes will be pushed once I know it at least builds.

Reply:
Do you know what specific files aren't needed, or can I just `cleanup` the whole `/usr/bin` folder?

Reply:
Thunderbird seems to build and run just fine with ICU.

Reply:
In your build directory there is a `files` directory where you can see that is left in there, you probably don't need any binaries but just remove things until it explodes.

Reply:
testing with `cleanup: ["/bin/*", "/sbin/*"]` first.